src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource_bicep.py

```diff
@@ -16,6 +16,10 @@
     _bicep_version_check_file_path,
 )
 from azure.cli.core.azclierror import InvalidTemplateError
+from azure.cli.core.mock import DummyCli
+
+cli_ctx = DummyCli()
+cli_ctx.config.set_value("bicep", "use_binary_from_path", "false")
```

Comment:
Is it worth adding tests for some of these new scenarios (reading of config, usage of env vars)?

Reply:
Yup, will add tests.